docs/source/theory/intro.rst

```diff
@@ -3,6 +3,27 @@
 Introduction
 ============
 
+Concepts
+--------
+
+Reference Trajectory
+""""""""""""""""""""
+
+ImpactX is an *s*-based beam dynamics code, evolving particles *relative* to a reference trajectory.
+
+A reference trajectory is favorable instead of, e.g., differences to the beam centroid, because:
```

Comment:
Before this line, I would add:  

"The reference trajectory plays an important role in ImpactX.  In addition to specifying the nominal machine orbit, it specifies the relationship between the local beam coordinate system and the global lab coordinate system.